### DISCUSSION TITLE
I start over.

### DISCUSSION THREAD

**Discussion by whoisclebs** [octobearhub/octobear.org#63] [Announcements]
Last year I wanted to create the octobear community to bring people all over the world together, but I couldn't help keep the community going. After many personal issues, I will be returning with the community. The discord server will be closed and we will keep communication here. Although there are some changes, as the focus is now on the Brazilian community. We still hope to see contributors from all countries.

**Comment by TenTraicion** [comment]
Hope to see something new and awesome. and i would like to request to keep the discord server open as we can discuss important things there. Also wish to work as a team. Thanks!

**Reply by whoisclebs** [reply]
Good to see you again. I hope we can create something great for the community.
I was thinking of changing the site to hugo or jekyll for easier maintenance.

What I have in mind is to create a common point for people, my great difficulty in getting new members is the language barrier, where I live for example there are few English speakers.

One way out might be to create small communities within octobear. An Indian community, a Brazilian community and many others, so we can unite these communities at octobear. What do you think?

**Reply by whoisclebs** [reply]
The demand to change site to hugo or jekyll is i18n easy use. But we can continue with nextjs too

**Reply by TenTraicion** [reply]
i can help you with nodejs, ejs... next js is a new thing for me.. but i would like to help wherever possible. 
about the communities, i like that idea. we can divide the community in two parts, one international and one local. user will have access to both community. that way it will be easier for the users.
